plugins/inline-row-separators/src/inline_row_separators_renderer.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Defines a renderer that renders dummy inputs like line breaks.
+ */
+
+import * as Blockly from 'blockly/core';
+
+/**
+ * A subclass of the geras RenderInfo that renders all dummy inputs like line
+ * breaks, forcing any following input to be rendered on a separate row, even
+ * when the block type definition has `"inputsInline": true`.
+ */
+export class RenderInfo extends Blockly.geras.RenderInfo {
```

Comment:
Since the logic in here is so minimal, it would be really nice if we could make this a decorator instead of a subclass. But I think the interface of `RenderInfo` is too large to make a decorator really practical :/

Reply:
Done! Using the mixin technique that Rachel suggested: https://www.typescriptlang.org/docs/handbook/mixins.html